cmd/export_account_signers.go

```diff
@@ -22,13 +22,13 @@ should be used in an initial data dump. In order to get account information with
 the export_ledger_entry_changes command.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		cmdLogger.SetLevel(logrus.InfoLevel)
-		endNum, strictExport, isTest, isFuture, extra, _, datastoreUrl := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
-		cmdLogger.StrictExport = strictExport
-		env := utils.GetEnvironmentDetails(isTest, isFuture, datastoreUrl)
+		commonArgs := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		cmdLogger.StrictExport = commonArgs.StrictExport
+		env := utils.GetEnvironmentDetails(commonArgs.IsTest, commonArgs.IsFuture, commonArgs.DatastorePath)
```

Comment:
not blocking for this PR: wdty about passing an env arg instead of `IsTest` and `IsFuture`? I think we'd need to plan the breaking change with Goldsky, but might be a better UX

Reply:
Yeah that'd be good. We need to refactor a lot of the params TBH 

[Hubble-386](https://stellarorg.atlassian.net/jira/software/c/projects/HUBBLE/boards/65/backlog?selectedIssue=HUBBLE-386) although it doesn't have a description lol